{
  "discovery": {
    "hostname": "host1.lab.devnetsandbox.local",
    "managementIpAddress": "10.10.20.83",
    "reachabilityStatus": "Success",
    "pingStatus": "SUCCESS",
    "snmpStatus": "SUCCESS",
    "cliStatus": "SUCCESS",
    "httpStatus": "NOT-PROVIDED",
    "netconfStatus": "NOT-PROVIDED",
    "inventoryCollectionStatus": "In Progress",
    "inventoryReachabilityStatus": "Reachable",
    "id": "b439a2f3-474c-47a7-9af1-cfc5e5646e99"
  },
  "device": {
    "description": "Cisco IOS Software, C3560E Software (C3560E-UNIVERSALK9-M), Version 15.2(1)E1, RELEASE SOFTWARE (fc2) Technical Support: http://www.cisco.com/techsupport Copyright (c) 1986-2013 by Cisco Systems, Inc. Compiled Fri 22-Nov-13 10:28 by prod_rel_team",
    "type": "Cisco Catalyst 3560X-24P Switch",
    "lastUpdated": "2021-05-31 19:08:17",
    "lineCardCount": "0",
    "lineCardId": "",
    "locationName": null,
    "managedAtleastOnce": true,
    "managementIpAddress": "10.10.20.83",
    "memorySize": "NA",
    "platformId": "WS-C3560X-24P-L",
    "reachabilityFailureReason": "",
    "reachabilityStatus": "Reachable",
    "snmpLocation": "",
    "tagCount": "0",
    "tunnelUdpPort": null,
    "uptimeSeconds": 23587444,
    "waasDeviceMode": null,
    "apEthernetMacAddress": null,
    "apManagerInterfaceIp": "",
    "associatedWlcIp": "",
    "bootDateTime": "2020-08-31 19:04:17",
    "collectionStatus": "Partial Collection Failure",
    "errorCode": "UNKNOWN",
    "errorDescription": "NCIM12024: All information from the device could not be collected successfully or the inventory collection for this device has not yet started. It may be a temporary problem that will resolve automatically. Resync the device, if that does not resolve the problem, please contact Cisco TAC.",
    "family": "Switches and Hubs",
    "interfaceCount": "0",
    "serialNumber": "FDO1642H0K0",
    "lastUpdateTime": 1622488097747,
    "inventoryStatusDetail": "<status><general code=\"UNKNOWN\" reason=\" Exception occured while trying to commit inventory transaction: org.springframework.orm.hibernate3.HibernateOptimisticLockingFailureException: Object of class [com.cisco.enc.apic.inventory.Host] with identifier [8531533]: optimistic locking failed; nested exception is org.hibernate.StaleObjectStateException: Row was updated or deleted by another transaction (or unsaved-value mapping was incorrect): [com.cisco.enc.apic.inventory.Host#8531533] deviceId: 8509504; CredentialId: 8509504; ManagementIP: 10.10.20.83\"/></status>",
    "managementState": "Managed",
    "collectionInterval": "Global Default",
    "deviceSupportLevel": "Unsupported",
    "softwareType": "IOS",
    "softwareVersion": "15.2(1)E1",
    "series": "Cisco Catalyst 3560-X Series Switches",
    "snmpContact": "",
    "roleSource": "AUTO",
    "location": null,
    "upTime": "273 days, 0:04:25.10",
    "hostname": "host1.lab.devnetsandbox.local",
    "role": "ACCESS",
    "macAddress": "ac:f2:c5:3d:7d:80",
    "instanceTenantId": "5f94b901d5a01000c622fbe4",
    "instanceUuid": "b439a2f3-474c-47a7-9af1-cfc5e5646e99",
    "id": "b439a2f3-474c-47a7-9af1-cfc5e5646e99"
  }
}